.github/workflows/nightly-ci.yml

```
@@ -354,6 +355,30 @@ jobs:
       image_tag_suffix: '-nightly'
     secrets: inherit
 
+  vllm-h100-test:
+    name: vllm-runtime # This name overlaps with other vllm jobs to group them in the UI
+    needs: [vllm-build, resolve-source-sha, compute-release-mode]
+    if: ${{ needs.compute-release-mode.outputs.run_tests == 'true' }}
+    uses: ./.github/workflows/shared-test.yml
+    with:
+      dd_env: nightly
+      dd_flaky_retry_enabled: 'false'
```

actionlint (1.7.12)

[error] input "dd_flaky_retry_enabled" is typed as string by reusable workflow "./.github/workflows/shared-test.yml". bool value cannot be assigned

(expression)